### PR TITLE
genconfig: add helper info

### PR DESCRIPTION
**Pull request by lxbsz**
Signed-off-by: Xiubo Li <xiubli@redhat.com>

### CODE REVIEW

**Comment by lxbsz** [comment]
@pkalever Please review.

**Comment by pkalever** [comment]
@lxbsz we wantedly didn't mention this command info at helper, as we wanted to make this available to Support teams only.

Now that, we have decided to make this mandate at upgrade paths, probably it make-sense to include this.

Can you also make changes at the gluster-block man page about the same ?

**Comment by pkalever** [comment]
@lxbsz appreciate the changes done at README.md, but my ask was to do changes in docs/gluster-block.8 (man page). Please add an usage example in the examples section.

Thanks!

File: cli/gluster-block.c [hunk 1]
```diff
@@ -245,6 +245,9 @@ glusterBlockHelp(void)
       "  replace <volname/blockname> <old-node> <new-node> [force]\n"
       "        replace operations.\n"
       "\n"
+      "  genconfig <volname[,volume2,volume3,...]> enable-tpg <host>\n"
+      "        generate target config operations.\n"
```
Comment (pkalever) [inline cli/gluster-block.c]: generate the block volumes target configuration.

File: README.md [hunk 2]
```diff
@@ -93,29 +93,38 @@ You can run gluster-blockd as systemd service, note '/etc/sysconfig/gluster-bloc
...
 usage:
   gluster-block <command> <volname[/blockname]> [<args>] [--json*]
 
 commands:
   create  <volname/blockname> [ha <count>]
                               [auth <enable|disable>]
                               [prealloc <full|no>]
+                              [storage <filename>]
+                              [ring-buffer <size-in-MB-units>]
                               <host1[,host2,...]> <size>
```
Comment (pkalever) [inline README.md]: As per the latest helper, this should be
<host1[,host2,...]> [size]

File: README.md [hunk 3]
```diff
@@ -93,29 +93,38 @@ You can run gluster-blockd as systemd service, note '/etc/sysconfig/gluster-bloc
...
 
-  modify  <volname/blockname> <auth enable|disable>
+  modify  <volname/blockname> [auth <enable|disable>] [size <size>] [force]
         modify block device.
 
+  replace <volname/blockname> <old-node> <new-node> [force]
+        replace operations.
+
+  genconfig <volname[,volume2,volume3,...]> enable-tpg <host>
+        generate target config operations.
```
Comment (pkalever) [inline README.md]: generate the block volumes target configuration.

**Comment by lxbsz** [comment]
@pkalever Yeah, all looks good to me, update them all.

File: README.md (hunk 2 above)
Comment (pkalever) [reply README.md]: @lxbsz you missed this request.

**Comment by pkalever** [comment]
Thanks @lxbsz

**Pull request merged**